osa/configs/sequencer.cfg

```diff
@@ -21,6 +21,7 @@ PEDESTAL_DIR: %(CALIB_BASE_DIR)s/drs4_baseline
 DL1_DIR: %(BASE)s/DL1
 DL1AB_DIR: %(BASE)s/DL1
 MUON_DIR: %(BASE)s/DL1
+INTERLEAVED_DIR: %(BASE)s/DL1
```

Comment:
According to Franca we should place the interleaved files in `.../DL1/interleaved/`subdirectory so this has to be reworked